### PR TITLE
Create Order-Returns-Handling-Calculations KB tutorial

### PR DESCRIPTION
**Pull request by maxwmaher**

### CODE REVIEW

**Comment by robertsoniv** [comment]
Hey @maxwmaher thank you for the content! Did you write this all in GitHub or did you run locally? Just want to make sure your new author token will work properly. If you wrote in GitHub I can run this PR locally to make sure.

**Comment by maxwmaher** [comment]
Hey, I wrote it locally in VS Code. Let me know if there's something I can do to verify it will work as intended.

**Comment by robertsoniv** [comment]
Alrighty. If you did it that way and the `npm start` didn't break then I think we're all good. I'll review the content asap.

**Comment by maxwmaher** [comment]
Ha I should probably try that to make sure. One moment...

File: content/documents/order-returns-handling-calculations.mdx [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+- A $10 line item-level promotion was applied to line X002, essentially dropping its list price from $60 to $50 (the value sent for tax estimation purposes).
+- Lastly, a $20 order promotion was applied to the full order. Because line X001 is not promotion eligible, our tax service will proportionally apply this promotion to lines X002 and X003.
+- $30 worth of promotions were applied to this order, causing the taxable subtotal to drop from $170 to $140.
+
+## Taxes & Promotions
+
+OrderCloud does not provide a calculation for how much an order-level promotion reduces the cost of a `LineItem` on the `Order`, allowing businesses to remain flexible on how they wish to apply these discounts. In our example, we've decided to lean on our tax service to apply the order-level promotion proportionally across eligible `LineItems` for us (6.67 for line X002 + 13.33 for line X003 = $20 order-level promotion). That contrasts with how we handled the line item-level promotion, and the reason for the verbose property names of `LineItemPromoAppliedToTaxableSubtotal` and `OrderLevelPromoNotAppliedToTaxableSubtotal`.
```
[the remark this answers is not in the document]
Comment (maxwmaher) [reply content/documents/order-returns-handling-calculations.mdx]: Thanks for reviewing, all three should be resolved now.

**Comment by maxwmaher** [comment]
Actually looks like I might need environment variables to run locally. Would one of you be able to reach out to me with those?

**Comment by djsteinmetz** [comment]
I gotchu

**Pull request merged**